openff/toolkit/topology/molecule.py

```diff
@@ -2909,6 +2909,7 @@ def assign_partial_charges(
         partial_charge_method,
         strict_n_conformers=False,
         use_conformers=None,
+        force_field=None,
```

Comment:
Thanks for looking into this, but I think this would introduce a lot of complexity/user surprises. So I think we should remove this in favor of only allowing users to get charges using `forcefield.get_partial_charges`.

Reply:
Note from @Yoshanuikabundi  during meeting: Add a note in the docstring pointing people to `get_partial_charges` if they do want to use a ff here.

Reply:
Done in 29fb897 and 43a27a6